packages/flutter_test/lib/src/test_text_input.dart

```diff
@@ -19,6 +22,19 @@ import 'widget_tester.dart';
 /// * [WidgetTester.showKeyboard], which uses this class to simulate showing the
 ///   popup keyboard and initializing its text.
 class TestTextInput {
+  /// Create a fake keyboard backend.
+  ///
+  /// The [onCleared] argument may be set to be notified of when the handle is
```

Comment:
This doc comment is not clear to me. What's the handle? How is it reset?

Reply:
changed to "keyboard is dismissed".